src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -1084,6 +1088,15 @@ public void setCopyReferences (boolean copyReferences) {
 		this.copyReferences = copyReferences;
 	}
 
+	/**
+	 * This method sets the the size of resolver maps when they are reset/cleared.
+	 *
+	 * @param ioimClearSize
+	 */
+	public void setIdenityObjectIntMapClearSize(int ioimClearSize) {

```

Comment:
This setting does not belong on the Kryo class, lest it become a junk drawer for various settings.

Why do users need this configurable? When would they want to change it?

Reply:
I'm against hardcoding values in arbitrary places. As such, I thought a constant would be nice alternative. Then I figured, well, if I'm making it a constant I might as well make it tunable. It is easy enough to go back to just a constant. I'd change the invocation paradigm a bit in this case though. I'll do the change and you'll see what I mean in this PR.